packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.tsx

```diff
@@ -23,8 +24,14 @@ export class DocumentCard extends React.Component<IDocumentCardProps, any> {
       };
     }
 
+    // if this element is actionable it should have an aria role
+    let role = actionable ? (onClick ? 'button' : 'link') : null;
+    let tabIndex = actionable ? 0 : null;
+
     return (
       <div
+        tabIndex={ tabIndex }
```

Comment:
I am going to apply your change and do npm link to see how it will impact current build. Since our feature is already 100% in Prod, we don't want to have regression.